core/payment-driver/zksync/src/zksync.rs

```diff
@@ -0,0 +1,53 @@
+// External uses
+pub use client::wallet::PackedEthSignature;
+use client::wallet::ETH_SIGN_MESSAGE;
+use tiny_keccak::keccak256;
+use web3::types::H160;
+
+// Workspace uses
+
+// Local uses
+use crate::utils::get_sign_tx;
+
+pub async fn get_zksync_seed(pub_address: H160) -> Vec<u8> {
+    info!("Creating zksync seed. address={}", pub_address);
+    let address = pub_address.as_bytes().into();
+    let sign_tx = get_sign_tx(address);
+    let seed = sign_tx(message_to_signed_bytes(ETH_SIGN_MESSAGE.as_bytes(), true)).await;
+    convert_to_eth_signature(seed)
+}
+
+pub async fn eth_sign_transfer(pub_address: H160, message: String) -> Vec<u8> {
+    info!(
+        "Signing eth transfer. address={}, message={}",
+        pub_address, message
+    );
+    let address: [u8; 20] = *pub_address.as_fixed_bytes();
+    let sign_tx = get_sign_tx(address.into());
+    let eth_sign_hex = sign_tx(message_to_signed_bytes(message.as_bytes(), true)).await;
+    convert_to_eth_signature(eth_sign_hex)
+}
+
+fn message_to_signed_bytes(msg: &[u8], include_prefix: bool) -> Vec<u8> {
```

Comment:
More like *singable* bytes. They're not yet signed but to be signed.

Reply:
good one! updated :)